widgets/src/api/pda/summary.js

```diff
@@ -0,0 +1,23 @@
+import makeRequest from 'api/makeRequest';
+import { DOMAINS, METHODS } from 'api/constants';
+import MOCK_SUMMARIES from 'mocks/summary/summaries';
+import MOCK_SUMMARY from 'mocks/summary/summary';
+
+export const getSummaries = async connection =>
+  makeRequest({
+    domain: DOMAINS.PDA,
+    uri: `/connections/${connection}/summaries`,
+    method: METHODS.GET,
+    mockResponse: MOCK_SUMMARIES,
+    useAuthentication: false,
```

Comment:
working locally, the API handles auth by itself, but when we put all online, we will need this to make our requests to PDA plugin

Reply:
useAuthentication now `true`